src/workloads/workloads.csproj

```diff
@@ -48,38 +48,32 @@
 
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
 
-  <ItemDefinitionGroup>
-    <!-- Visual Studio components must be versioned. Build tasks will fall back to cobbling a version number from
-         the manifest information unless it's overridden. -->
-    <ComponentResources Version="$(FileVersion)" />
-  </ItemDefinitionGroup>
-
   <Target Name="Build" DependsOnTargets="GetAssemblyVersion;_GetVersionProps;_GenerateMsiVersionString">
     <ItemGroup>
       <!-- Overrides for Visual Studio setup generation. If the workload definition IDs change,
            these must be updated. -->
       <ComponentResources Include="microsoft-net-runtime-mono-tooling" Title=".NET Shared Mobile Build Tools"
-                          Description="Shared build tasks for mobile platform development."/>
+                          Description="Shared build tasks for mobile platform development." Version="$(FileVersion)"/>
```

Comment:
Can this be a line at the end that's just `<ComponentResources Include="@(ComponentResources)" Version="$(FileVersion)"/>`

Reply:
We could do a temp item and then assign, otherwise we end up with a duplicate list I think